dev-lang/clojure/clojure-1.10.3.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="test"
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="General-purpose programming language with an emphasis on functional programming"
+HOMEPAGE="https://clojure.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${P}.tar.gz"
+
+LICENSE="EPL-1.0 Apache-2.0 BSD"
+SLOT="1.10.3"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+RESTRICT="!test? ( test )" # broken due to file not found issue and more
+
+CDEPEND="
+	dev-java/spec-alpha:0.2
+	dev-java/core-specs-alpha:0.2
+	dev-java/ant-core:0"
+
+RDEPEND="
+	${CDEPEND}
+	>=virtual/jre-1.8"
+
+DEPEND="
+	${CDEPEND}
+	>=virtual/jdk-1.8"
```

Comment:
Please switch from `CDEPEND` to something else, like `COMMON_DEPEND`.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Avoid_using_single-char_.2ADEPEND_variable